<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventMesPageWidget</class>
 <widget class="QWidget" name="EventMesPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>810</width>
        <height>470</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="0" colspan="3">
          <widget class="QGroupBox" name="grpRelatedEvents">
           <property name="title">
            <string>Стандарты и диагнозы других отделений</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="CTableView" name="tblRelatedEvents">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblMes">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>МЭС</string>
           </property>
           <property name="buddy">
            <cstring>cmbMes</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="chkHTG">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>ВМП</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblMesSpecification">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Особенности выполнения МЭС</string>
           </property>
           <property name="buddy">
            <cstring>cmbMesSpecification</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btnCheckMes">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Проверить выполнение стандарта</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="btnShowMes">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Показать требования стандарта</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="CRBComboBox" name="cmbMesSpecification">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="CRBComboBox" name="cmbHTG">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <layout class="QHBoxLayout" name="MESLayout">
           <item>
            <widget class="CMESComboBox" name="cmbMes"/>
           </item>
          </layout>
         </item>
         <item row="5" column="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>CMESComboBox</class>
   <extends>QComboBox</extends>
   <header>library/MES/MESComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
